src/publishProject/publishProjectWebViewController.ts

```diff
@@ -286,8 +317,12 @@ export class PublishProjectWebViewController extends FormWebviewController<
 
                 try {
                     const databaseName = state.formState.databaseName || projectName;
-                    // TODO: Build connection string from connection details when server/database selection is implemented
-                    const connectionString = "";
+                    // Connection string depends on publish target:
+                    // - For container targets: empty string (no server connection)
```

Comment:
Why do container targets have no connection string?  Is that because we're provisioning a new container (and _then_ would use a connection string for that new container)?  Can you add that to this comment?